tests/CMakeLists.txt

```diff
@@ -3,3 +3,6 @@ add_subdirectory_if_kconfig(ztest)
 zephyr_include_directories_ifdef(CONFIG_TEST
   $ENV{ZEPHYR_BASE}/tests/include
   )
+
+zephyr_syscall_include_ifdef(CONFIG_EXECUTION_BENCHMARKING
+  "${CMAKE_CURRENT_SOURCE_DIR}/benchmarks/timing_info/src")
```

Comment:
why is this here?
this should go in the CMakeLists.txt for the specific timing_info test!